.github/workflows/check.yml

```
@@ -1,7 +1,9 @@
 name: Build zola website to ensure no errors
 
 on:
-  pull_request
+  pull_request:
+    branch:
+      master
```

The syntax is correct but now that `master` has been renamed to `main` this PR needs an update too.

```suggestion
      main
```